### ISSUE TITLE
0.5.0 is not packaged in Homebrew

### ISSUE THREAD

**Issue by JensRantil** [bazelbuild/bazel#3107]
**Background:** According to https://bazel.build/versions/master/docs/install.html latest stable version is 0.5.0. According to https://bazel.build/versions/master/docs/install-os-x.html, the preferred way of installed Bazel is through Homebrew.

**Problem:** Homebrew does not package version 0.5.0. From my local laptop:
```bash
$ brew update
Already up-to-date.
$ brew upgrade bazel
Error: bazel 0.4.5 already installed
$ bazel version
Build label: 0.4.5-homebrew
Build target: bazel-out/local-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Thu Mar 16 13:39:33 2017 (1489671573)
Build timestamp: 1489671573
Build timestamp as int: 1489671573
```

**Expected:** That 0.5.0 is accessible through Homebrew.

**Workarounds:** Using a less recommended installation listed on https://bazel.build/versions/master/docs/install-os-x.html.

**Comment by laszlocsomor** [comment]
@mhlopko : Do you know why our Homebrew package is out of date?

**Comment by raindev** [comment]
The reason for the rever is bazelbuild/bazel#3063 which is going to be fixed with 0.5.1. See: https://github.com/Homebrew/homebrew-core/commit/1f2198ade252bce7a9e5c993eebad70690daa1e8

**Comment by mhlopko** [comment]
Just like @raindev is saying :) There's a macos regression in 0.5.0 which will be fixed in 0.5.1. We'll update homebrew recipe to 0.5.1 directly.

**Comment by mhlopko** [comment]
The pull request for 0.5.1 was just merged in homebrew. It should serve bazel 0.5.1 from now on.
